src/include/extern.h

```diff
@@ -2534,7 +2534,7 @@ static WT_INLINE uint64_t __wt_clock_to_nsec(uint64_t end, uint64_t begin)
 static WT_INLINE uint64_t __wt_rdtsc(void) WT_GCC_FUNC_DECL_ATTRIBUTE((warn_unused_result));
 static WT_INLINE uint64_t __wt_safe_sub(uint64_t v1, uint64_t v2)
   WT_GCC_FUNC_DECL_ATTRIBUTE((warn_unused_result));
-static WT_INLINE uint64_t __wt_txn_id_alloc(WT_SESSION_IMPL *session, bool publish)
+static WT_INLINE uint64_t __wt_txn_id_alloc(WT_SESSION_IMPL *session, bool release)
```

Comment:
I think publish is used for a different context here. I don't think we should use release here. It's misleading.

Reply:
This one is a bit funky, in `__wt_txn_id_alloc` it does determine whether we do 4 subsequent release writes. We could make it publish but then the comment above those release writes may need changing. What do you think about that?

Reply:
I think we then change the comment?

Reply:
aca63526d